pkg/apis/trust/v1alpha1/types_bundle.go

```diff
@@ -76,6 +76,10 @@ type BundleSource struct {
 	// InLine is a simple string to append as the source data.
 	// +optional
 	InLine *string `json:"inLine,omitempty"`
+
+	// CCADB is an embedded copy of the certificates in Mozilla's root store
+	// that have the websites trust bit enabled.
+	CCADB *bool `json:"ccadb,omitempty"`
```

Comment:
I `*string` typed field with e.g. `publicTrustBundle: CCADB` allows for further extensibility in future - not sure on exact field naming, but the Kubernetes API convention calls out to be careful with bool typed fields (see the final bullet point here https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md#primitive-types)

Reply:
👍 makes sense

Reply:
Would it be worth considering a different name to "CCADB"? My instinct is that a user would have no idea what a "CCADB" is and wouldn't find the name very easy to remember.

`mozilla` would match closer to what I'd probably expect, or maybe something like `embedded` to reflect that this is a trust store embedded in the `trust` tool.